lib/pages/view-post-page.js

```diff
@@ -68,7 +68,7 @@ export default class ViewPostPage extends BaseContainer {
 
 	commentEventuallyShown( comment ) {
 		const self = this;
-		const commentSelector = By.xpath( `//p[text() = "${comment}"]` );
+		const commentSelector = By.css( `p:contains("${comment}")` );
```

Comment:
Yes work on it for sure.

We usually add an attribute with the prefix `data-e2e-` to make it clear what it's for.

See this file for an example: https://github.com/Automattic/wp-calypso/blob/324c932bbce4c042659d8354a6a03f851455eb27/client/components/domains/domain-suggestion/index.jsx#L36